jupyterlab_comments/.prettierignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
This might be worth moving to the root of the repository. By default I think it ignores node_modules and compiled sources, but it is irritating to get warnings when you save a package.json file.

Reply:
This file was from an old cookie cutter template, so I think I'm just going to delete it